src/unix/notbsd/android/mod.rs

```diff
@@ -95,6 +95,10 @@ s! {
         #[cfg(target_pointer_width = "32")]
         __bits: [__CPU_BITTYPE; 1],
     }
+
+    pub struct sem_t {
+        pub count: ::c_uint,
```

Comment:
Perhaps this could be opaque like the other platforms? That'd mean the special case in build.rs could be removed as well